### PR TITLE
Address yarn lints

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

Couple of lint issues that came up in: https://github.com/MetaMask/metamask-mobile/pull/2523.

### CODE REVIEW

File: app/components/Views/Entry/index.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ const Entry = props => {
 
 	const animation = useRef(null);
 	const animationName = useRef(null);
-	const opacity = new Animated.Value(1);
+	const opacity = useMemo(() => new Animated.Value(1), []);
```
Comment (wachunei) [inline app/components/Views/Entry/index.js:79]: I'd use `useRef` here instead:

Example:
https://github.com/MetaMask/metamask-mobile/blob/develop/app/components/UI/Swaps/components/LoadingAnimation/index.js#L120

**Comment by rickycodes** [comment]
I was trying to figure out what caused these to crop up in the first place and I realized it was an upgrade to `eslint`... that upgrade has now also been included here.

File: .eslintrc.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-commonjs
```
Comment (rickycodes) [inline .eslintrc.js:1]: addresses:
```
/home/ricky/projects/metamask-mobile/.eslintrc.js
  1:1  error  Expected "export" or "export default"  import/no-commonjs
  ```

File: app/__mocks__/rn-fetch-blob.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ export default {
 	config: noop,
 	session: noop,
 	fs: {
+		writeFile: () => Promise.resolve(),
```
Comment (rickycodes) [inline app/__mocks__/rn-fetch-blob.js:12]: this change fixes this mock, I guess it broke in the upgrade?

File: app/components/UI/SignatureRequest/index.js [hunk 4]
```diff
@@ -232,8 +232,9 @@ class SignatureRequest extends PureComponent {
 		let expandedHeight;
 		if (Device.isMediumDevice()) {
 			expandedHeight = styles.expandedHeight2;
-		} else if (type === 'ethSign' && Device.isMediumDevice()) {
-			expandedHeight = styles.expandedHeight1;
+			if (type === 'ethSign') {
```
Comment (rickycodes) [inline app/components/UI/SignatureRequest/index.js:235]: addresses:
```
/home/ricky/projects/metamask-mobile/app/components/UI/SignatureRequest/index.js
  235:14  error  This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain  no-dupe-else-if
  ```

**Review by wachunei — APPROVED** [review]

**Review by andrepimenta — APPROVED** [review]

**Pull request merged**